source/fundamentals/crud/read-operations/distinct.txt

```diff
@@ -0,0 +1,161 @@
+.. _node-fundamentals-distinct:
+
+========================
+Retrieve Distinct Values
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Use the ``distinct()`` method to retrieve all distinct values for a specified field 
+across a collection.
+
+Sample Documents
+~~~~~~~~~~~~~~~~
+
+Follow the instructions in the examples below to insert data into a collection and
+return the unique values of a given field. Consider the following collection of 
+documents that describe restaurants: 
+
+.. code-block:: json
+
+   [
+      { "_id": 1, "restaurant": "White Bear", "borough": "Queens", "cuisine": "Chinese" },
+      { "_id": 2, "restaurant": "Via Carota", "borough": "Manhattan", "cuisine": "Italian" },
+      { "_id": 3, "restaurant": "Borgatti's", "borough": "Bronx", "cuisine": "Italian" },
+      { "_id": 4, "restaurant": "Tanoreen", "borough": "Brooklyn", "cuisine": "Middle Eastern" },
+      { "_id": 5, "restaurant": "Äpfel", "borough": "Queens", "cuisine": "German" },
+      { "_id": 6, "restaurant": "Samba Kitchen", "borough": "Manhattan", "cuisine": "Brazilian" },
+   ]
+
+.. include:: /includes/access-cursor-note.rst
+
+Distinct
+--------
+
+Use a document field name as a parameter for the ``distinct()`` method to return a 
+list of the field's unique values. Optionally, you can specify a ``query`` 
+parameter to narrow down your results or an ``options`` parameter to set the 
+string comparison language.
```

Comment:
issue (non-blocking): The way this reads to me, it feels like an "OR" decision. It might be more clear to state that this is additive.

```suggestion
list of the field's unique values. You can also specify an optional ``query`` 
parameter to narrow your results and an optional``options`` parameter to set the 
string comparison language.
```